modules/virt-cloud-platforms.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="virt-cloud-platforms_{context}"]
+= Compatible cloud platforms
+
+{VirtProductName} is compatible with a variety of public cloud platforms. Each cloud platform has specific storage provider options available. The following table outlines which platforms are fully supported (GA) and which are currently offered as Technology Preview features.
+
+:FeatureName: Installing {VirtProductName} on certain cloud platforms
+include::snippets/technology-preview.adoc[]
+
+[cols="1,1,2", options="header"]
+|===
+| Vendor
+| Status
+| Storage
+
+| {aws-first}
+| GA
+| Elastic Block Store (EBS), {odf-first}, Portworx, FSx (NetApp)
+
+| {product-rosa}
+| GA
+| EBS, Portworx, FSx (Q3), {odf-short}
+
+| {oci-first-no-rt}
+| Technology Preview
+| {oci} native storage
+
+| Azure Red{nbsp}Hat OpenShift (ARO)
+| Technology Preview
+| {odf-short}
+
+| {gcp-first}
+| Technology Preview
+| {gcp-short} native storage
+
+| {product-dedicated}
+| Technology Preview
+| {gcp-short} native storage
+
+// | IBM Cloud
+// | Technology Preview
+// | IBM Fusion
+|===
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://github.com/oracle-quickstart/oci-openshift/blob/main/custom_manifests/oci-ccm-csi-drivers/v1.30.0-RWX-LA/openshift-virtualization.md[Installing {VirtProductName} on OCI]
+* link:https://access.redhat.com/articles/7118050[{VirtProductName} and Oracle Cloud Infrastructure known issues and limitations]
```

Comment:
In  addition to our kbase, there is the oracle documentation in https://github.com/oracle-quickstart/oci-openshift/blob/main/custom_manifests/oci-ccm-csi-drivers/v1.30.0-RWX-LA/openshift-virtualization.md#installation

Reply:
I linked to the top of this page in the previous bullet point :)